projects/flow.groups-2D-random-walk/src/init.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+"""Initialize the project's data space.
+
+Iterates over all defined state points and initializes
+the associated job workspace directories."""
+import logging
+
+import numpy as np
+import signac
+
+logger = logging.getLogger()
+# Lower level to prevent output
+logger.setLevel(logging.WARN)
+
+N_STD = 20
+# The different standard deviations to use for generating moves in the random
+# walk
+STANDARD_DEVIATIONS = np.linspace(0.1, 1, N_STD)
+NUMBER_REPLICAS = 100
+RUN_N_STEPS = 5_000
```

Comment:
No need to define a variable for `N_STD`, the keyword arguments are clear enough. Also `RUN_N_STEPS` doesn't need the `N`.
```suggestion
# The different standard deviations to use for generating moves in the random
# walk
STANDARD_DEVIATIONS = np.linspace(start=0.1, stop=1, num=20)
NUMBER_REPLICAS = 100
RUN_STEPS = 5_000
```